build_backend.py

```
@@ -25,6 +27,360 @@
+    build-time deps in best-effort mode.
+    """
+    cuda_major = _detect_cuda_major()
+    wheels: list[str] = []
+    if built_nixl:
+        wheels.append(f"nixl-cu{cuda_major}>=1.0.1")
+    if built_nccl:
+        wheels.append(f"nvidia-nccl-cu{cuda_major}>=2.30.4")
+    if not wheels:
+        return
+    print(f"[BUILD_NVEP] pip install --no-deps {' '.join(wheels)}")
+    subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)